content/notebooks/data_discovery_and_access/data_discovery_and_access.ipynb

```diff
@@ -289,9 +304,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Downloading Files Locally (not recommended)\n",
+    "## Downloading Files (not recommended)\n",
+    "\n",
+    "Though it is **not recommended**, there may be instances where data files must be downloaded for certain specific science cases. To do that, we can use the URIs and the `S3FileSystem.get` function (documentation [here](https://s3fs.readthedocs.io/en/latest/api.html#s3fs.core.S3FileSystem.get)). Running the below cell will download the data to your personal instance of the science platform, however the same code could be combined with some of the above cells and run on your local machine to download the data to a private computer.\n",
     "\n",
-    "Though it is **not recommended**, there may be instances where data files must be downloaded locally for certain specific science cases. To do that, we can use the URIs and the `S3FileSystem.get` function (documentation [here](https://s3fs.readthedocs.io/en/latest/api.html#s3fs.core.S3FileSystem.get))."
+    "*NOTE*: MAST data can be downloaded on your private computer using `anon=True` in the `S3FileSystem` initialization. However to download the preliminary sample of Roman data, you will need to set up the credentials correctly. This is done for you on the science platform, but would need to be manually set on a private computer."
```

Comment:
Users cannot download the Roman simulated data from the S3 bucket to their private computer. I confirmed with Octarine that the S3 bucket is currently set to not be visible unless you are on the platform, no matter what authentication a user may have. So the second and third sentences need to be changed:

"However, the preliminary, simulated sample of Roman data on the science platform are currently not accessible outside of the science platform."